src/themes.js

```diff
@@ -43,6 +43,8 @@ const defaultTheme = {
   maxWidth: '832px',
   transition: '0.3s cubic-bezier(1, -0.65, 0, 2.25)',
   transitionEase: '0.3s ease-in-out',
+  transitionEaseOut:
+    ' 0.3s cubic-bezier(0.175, 1.03, 0.32, 1.366),padding 0.3s ease, margin 0.3s ease',
```

Comment:
There is an extra space before the `0.3s` and you don't need to add `,padding 0.3s ease, margin 0.3s ease`. And you could probably round the cubic-bezier, like `0.175, 1, 0.3, 1.35` :)